src/endpoints/dto/create-endpoint.dto.ts

```diff
@@ -28,6 +34,10 @@ export class CreateEndpointDto {
   @ApiProperty()
   route: string;
 
+  @IsString()
+  @ApiProperty()
+  contentType?: string;
```

Comment:
I feel like contenttype should be an enum 

You can check this link for most common content types: https://stackoverflow.com/questions/23714383/what-are-all-the-possible-values-for-http-content-type-header

Reply:
You are right.., I forgot to consider that... Thanks Azeez.